<?xml version="1.0"?>
<launch>
    <node pkg="mdr_hand_over_action" type="hand_over_action" name="hand_over_server" output="screen">
        <param name="move_arm_server" value="move_arm_server" />
        <param name="gripper_controller_pkg_name" value="mas_hsr_gripper_controller" />
        <param name="init_config_name" value="neutral" />
        <param name="hand_over_dmp" value="$(find mdr_hand_over_action)/config/trajectory_weights/grasp.yaml" />
        <param name="dmp_tau" value="30" />
    </node>
</launch>
